utils/nctl/sh/contracts/auction/do_bid.sh

```diff
@@ -1,100 +1,100 @@
 #!/usr/bin/env bash
-#
-# Invokes auction.add-bid entry point.
-# Globals:
-#   NCTL - path to nctl home directory.
-# Arguments:
-#   Network ordinal identifier (optional).
-#   Node ordinal identifier (optional).
-#   User ordinal identifier (optional).
-#   Bid amount (optional).
-#   Delegation rate (optional).
-#   Gas price (optional).
-#   Gas payment (optional).
 
 #######################################
 # Imports
 #######################################
 
-# Import utils.
 source $NCTL/sh/utils.sh
 
+#######################################
+# Submits an auction bid.
+# Arguments:
+#   Network ordinal identifier.
+#   Node ordinal identifier.
+#   Validator ordinal identifier.
+#   Bid amount.
+#   Delegation rate.
+#   Gas price.
+#   Gas payment.
+#######################################
+function do_auction_bid_submit()
+{
+    local NET_ID=${1}
+    local NODE_ID=${2}
+    local BIDDER_ID=${3}
+    local AMOUNT=${4}
+    local DELEGATION_RATE=${5}
+    local GAS=${6}
+    local PAYMENT=${7}
+
+    local BIDDER_PUBLIC_KEY=$(get_account_key $NET_ID $NCTL_ACCOUNT_TYPE_NODE $BIDDER_ID)
+    local BIDDER_SECRET_KEY=$(get_path_to_secret_key $NET_ID $NCTL_ACCOUNT_TYPE_NODE $BIDDER_ID)
+    local NODE_ADDRESS=$(get_node_address_rpc $NET_ID $NODE_ID)
+    local PATH_TO_CONTRACT=$(get_path_to_contract $NET_ID "add_bid.wasm")
+
+    log "dispatching deploy -> add_bid.wasm"
+    log "... network = $NET_ID"
+    log "... node = $NODE_ID"
+    log "... node address = $NODE_ADDRESS"
+    log "... contract = $PATH_TO_CONTRACT"
+    log "... bidder id = $BIDDER_ID"
+    log "... bidder secret key = $BIDDER_SECRET_KEY"
+    log "... bid amount = $amount"
```

Comment:
Minor nitpick: this uses the old variable name, so this produces nothing, but this is only relevant to logging, the core logic uses the correct new variable name.